### PR TITLE
Add "debouncing" to buttons 

### PR DESCRIPTION
**Pull request by britannio**
Partially addresses https://github.com/WorldHealthOrganization/app/issues/1219

Adds debouncing to the `PageButton` used in onboarding and creates a new button that'll hopefully be able to fully address #1219  and #1156 when implemented throughout the app. I'll leave that as a separate PR as it'll be a large refactor.

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).

- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR?
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by britannio** [comment]
The Policy Check action is broken https://github.com/WorldHealthOrganization/app/runs/668332590

File: client/flutter/lib/components/page_button.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/cupertino.dart';
 import 'package:who_app/constants.dart';
 
-class PageButton extends StatelessWidget {
+class PageButton extends StatefulWidget {
```
Comment (hspinks) [inline client/flutter/lib/components/page_button.dart:4]: Question: do we even still want the PageButton widget? It looks like it's currently only used in the onboarding flow on pages that are about to be removed/updated anyway. My suggestion would be that we do away with it and use the new Button widget wherever possible

Comment (britannio) [reply client/flutter/lib/components/page_button.dart:4]: Okay, should I wait until it's removed from onboarding or should I remove it in this PR? Doing the latter would give a use case for the new `Button` widget but I don't want to cause problems if changes are being undergone.

Comment (advayDev1) [reply client/flutter/lib/components/page_button.dart:4]: I’m using it in S-C

Comment (hspinks) [reply client/flutter/lib/components/page_button.dart:4]: Ah ok, didn't see that until tonight @advayDev1. Let's leave it in for now, we can refactor all buttons to use the new Button widget post v1. Marking this as resolved.

**Comment by hspinks** [comment]
> The Policy Check action is broken https://github.com/WorldHealthOrganization/app/runs/668332590

@britannio pulling in latest `master` will fix as of https://github.com/WorldHealthOrganization/app/commit/6e8661cdc786f70e0fff8880e4b9a3c139ee0fde

**Pull request merged**